chall02/gmolin.c

```diff
@@ -0,0 +1,33 @@
+/* ************************************************************************** */
+/*                                                                            */
+/*                                                        :::      ::::::::   */
+/*   gmolin.c                                           :+:      :+:    :+:   */
+/*                                                    +:+ +:+         +:+     */
+/*   By: gmolin <gmolin@student.hive.fi>            +#+  +:+       +#+        */
+/*                                                +#+#+#+#+#+   +#+           */
+/*   Created: 2020/03/30 16:04:38 by gmolin            #+#    #+#             */
+/*   Updated: 2020/03/30 16:36:40 by gmolin           ###   ########.fr       */
+/*                                                                            */
+/* ************************************************************************** */
+
+#include <stdlib.h>
+#include <stdio.h>
+
+char		*hv_rgb2hex(int r, int g, int b)
+{
+	char	*str;
+	int		array[3] = {r, g, b};
+	int 	i;
+
+	i = 0;
+	while (i < 3)
+	{
+		if (array[i] < 0 || array[i] > 255)
+			return (NULL);
+		i++;
+	}
```

Comment:
quite funny way to check :) 

It would have been clearer to create a small function `int is_in_range(int c)` but I like the idea.